plugins/org.jkiss.dbeaver.erd.model/src/org/jkiss/dbeaver/erd/model/ERDAssociation.java

```diff
@@ -103,8 +103,8 @@ protected void resolveAttributes(DBSEntityReferrer association, ERDEntity source
                         DBSEntityAttribute targetAttr = ((DBSTableForeignKeyColumn) attrRef).getReferencedColumn();
                         DBSEntityAttribute sourceAttr = attrRef.getAttribute();
                         if (sourceAttr != null && targetAttr != null) {
```

Comment:
Let's just change getAttributeByModel logic.

Reply:
I would avoid modification of current method because it impacted VQB and other places, I propose new one for use only in the ERD.